src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/TestUtil.java

```diff
@@ -40,6 +44,15 @@
 
 public final class TestUtil {
 
+    /**
+     * The stack size used in {@link #executeWithLimitedStackSize(Executable)}.
+     * This value should be as small as possible.
+     *
+     * @see <a href="https://www.baeldung.com/jvm-configure-stack-sizes">
+     *      Configuring Stack Sizes in the JVM</a>
+     */
+    private static final int MINIMAL_STACK_SIZE = 144 << 10;
```

Comment:
please explain "144 << 10"
why "<<" is used ?
please provide more details on this ...

Reply:
https://teamcity.jetbrains.com/viewLog.html?buildId=3291117&tab=Inspection&buildTypeId=Checkstyle_IdeaInspectionsPullRequest
![image](https://user-images.githubusercontent.com/900805/106378138-39814100-63b3-11eb-9664-6e7c5adcf75c.png)

I don't think that this inspection is useful. `144 * 1024` is clearly states that this is 144k. Should we ignore this IDEA inspection?